File: C/Condicional/Notas.c
#include <stdio.h>
int main()
{
    double nota1, nota2, notaFinal;

    printf("Digite a Primeira nota:");
    scanf("%lf", &nota1);

    printf("Digite a Segunda nota:");
    scanf("%lf", &nota2);

    notaFinal = nota1 + nota2;

    printf("NOTA FINAL = %.1lf\n", notaFinal);

    if (notaFinal < 60.0){
        printf("REPROVADO\n");



    }

}

File: Phyton/Condicional/Notas.py
nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))

notaFinal = nota1 + nota2

print(f"NOTA FINAL = {notaFinal:.1f}")
if notaFinal < 60.00:
    print("REPROVADO")
